util/options.cc

```diff
@@ -647,7 +647,9 @@ font_options_t::get_font (void) const
   if (!font_file)
     fail (true, "No font file set");
 
-  if (0 == strcmp (font_file, "-")) {
+  if (0 == strcmp (font_file, "-") ||
+      0 == strcmp (font_file, "/dev/stdin") ||
+      0 == strcasecmp (font_file, "STDIN")) {
```

Comment:
Not sure to support this always but as `stdin` and `STDIN` are both valid on DOS, I used `strcasecmp` instead.